index.html

```diff
@@ -10,8 +10,18 @@
 </head>
 <body class="container">
   <header class="head">
+    <div id="menu" class="hide hide-desktop">
+      <img src="assets/images/Icon.png" class="close-btn">
+      <ul>
+        <li><a href="#works" class="top">Works</a></li>
+        <li><a href="#about" class="top">About</a></li>
+        <li><a href="#contact" class="top">Contact</a></li>
+      </ul>
+    </div>
```

Comment:
- [ ] Nice work implementing the mobile menu, it look really good :clap: :sunglasses: However the `color` and the `font-size` are different compared to the figma design. Kindly change the `color` to `#3A4A42` and the `font-size` to `32px` to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-05-16 09-36-10](https://github.com/aamir-asaram/Portfolio/assets/91142432/66e90e3e-4d9c-4205-b831-3d8276435390)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-05-16 09-35-43](https://github.com/aamir-asaram/Portfolio/assets/91142432/f6d28e7c-5a15-4dee-8c6f-7038baab6387)